### ISSUE TITLE
multiple/duplicated smeltery recipes in JEI with enderio tic

### ISSUE THREAD

**Issue by TheSuspect2306** [SleepyTrousers/EnderIO#5329]
#### What happens:

if i go into JEI and search for signalum for the tconstruct smeltery it shows like 3 time of the same recipe instead of one, same for other alloys from thermal foundation

#### What you expected to happen:

instead of multiple/duplicated recipes just one recipe

#### Steps to reproduce:

1. i have installed tinkers construct, enderio with the enderio tconstruct addon and some other addons like thermal *
2. just look on the recipe for signalum or other other medals like electrum, the recipe is 3 times in JEI instead of one time, so 2 times too much, if i remove the enderio tconstruct addon its again only one time in JEI
...

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: 5.1.55
- EnderCore: 0.5.73
- Minecraft: 1.12.2
- Forge: 14.23.6.2847
- SpongeForge? no
- Optifine? no
- Single Player and Server

#### Your most recent log file where the issue was present: 

[pastebin/gist/etc link here]

do i need to put a logfile in when there is no crash or something ?

**Comment by HenryLoenwind** [comment]
Is this a request for a config flag to disable registering the Alloy Smelter recipes with TiC?

**Comment by TheSuspect2306** [comment]
i dont think so, the only problem ist that i only get 3 times the recipes in JEI when i have the EnderIO TinkerConstruct addon activate, so i dont know what you mean

**Comment by HenryLoenwind** [comment]
Ok, so your issue is that TiC doesn't de-duplicate recipes it gets from different mods. For that you'd need to talk to the TiC people. We only send our recipes to TiC, we have no control over what TiC then does with them.

**Comment by TheSuspect2306** [comment]
ok i try it on tconstruct, thanks for help :D

**Comment by TheSuspect2306** [comment]
hey, i got an answer from the tconstruct author or moderator, can i paste the url here ?

**Comment by TheSuspect2306** [comment]
still there ?
